apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -1,36 +1,109 @@
 import { html } from 'lit';
-
+import { ifDefined } from 'lit/directives/if-defined.js';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-checkbox';
 import { CheckboxProps } from '@justeattakeaway/pie-checkbox';
 /* eslint-enable import/no-duplicates */
 
+import { action } from '@storybook/addon-actions';
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
 type CheckboxStoryMeta = StoryMeta<CheckboxProps>;
 
-const defaultArgs: CheckboxProps = {};
+const defaultArgs: CheckboxProps = {
+    value: '',
+    name: '',
+    label: 'Label',
+    disabled: false,
+    checked: false,
+    indeterminate: false,
+};
 
 const checkboxStoryMeta: CheckboxStoryMeta = {
     title: 'Checkbox',
     component: 'pie-checkbox',
-    argTypes: {},
+    argTypes: {
+        value: {
+            description: 'The value of the checkbox (used as a key/value pair in HTML forms with `name`).',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        name: {
+            description: 'The name of the checkbox (used as a key/value pair with `value`). This is required in order to work properly with forms.',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        label: {
+            description: 'The visible label for the checkbox',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        checked: {
+            description: 'Indicates whether or not the checkbox is checked by default (when the page loads).',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        disabled: {
+            description: 'If true, disables the checkbox.',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        indeterminate: {
+            description: 'Indicates whether the checkbox visually shows a horizontal line in the box instead of a check/tick. It has no impact on whether the checkbox\'s value is used in a form submission. That is decided by the checked state, regardless of the indeterminate state.',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+    },
     args: defaultArgs,
     parameters: {
+        layout: 'centered',
```

Comment:
Out of interest – how does this visually change the preview (compared with the default?)

Reply:
Actually I checked that this parameter doesn't change the component's layout. For icon stories it centers the icon vertically but checkbox stays centered even without it